src/screens/ContactScreen/index.js

```diff
@@ -37,7 +38,9 @@ const ContactScreen = ({navigation}) => {
   const [selectedUsers, setSelectedUsers] = React.useState([]);
   const [isSearchMode, setIsSearchMode] = React.useState(false);
   const [isLoadingSearchResult, setIsLoadingSearchResult] = React.useState(false);
-  const {createSignChat, loadingCreateChat} = useCreateChat();
+  const {createSignChat, createAnonymousChat, loadingCreateChat} = useCreateChat();
+  const route = useRoute();
+  const {from: sourceScreen} = route.params;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `createAnonymousChat` function is used without any error handling. If this function throws an error, it will cause the application to crash. It would be better to wrap it in a try-catch block or ensure that it always handles errors internally and never throws them.
```diff
- createAnonymousChat(dataSelected);
+ try {
+   await createAnonymousChat(dataSelected);
+ } catch (error) {
+   console.error('Error creating anonymous chat:', error);
+ }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->